packages/mdc-menu/foundation.ts

```diff
@@ -120,30 +106,30 @@ export class MDCMenuFoundation extends MDCFoundation<MDCMenuAdapter> {
   }
 
   handleMenuSurfaceOpened() {
-    this.focusItemAtIndex_(this.defaultFocusItemIndex_);
+    switch (this.defaultFocusState_) {
+      case DefaultFocusState.LIST_ROOT:
+        this.adapter_.focusListRoot();
+        break;
+      case DefaultFocusState.FIRST_ITEM:
+        this.adapter_.focusItemAtIndex(0);
+        break;
+      case DefaultFocusState.LAST_ITEM:
+        this.adapter_.focusItemAtIndex(this.adapter_.getMenuItemCount() - 1);
+        break;
+      case DefaultFocusState.NONE:
+      default:
```

Comment:
do you need this default since it does nothing? Or should the `this.adapter_.focusListRoot();` go here instead?

Reply:
Good catch. Done!